rocAL_pybind/amd/rocal/readers.py

```diff
@@ -223,7 +223,11 @@ def video(sequence_length, file_list_frame_num=False, file_root="", image_type=t
         "file_list_frame_num": file_list_frame_num}  # VideoMetaDataReader
     b.videoMetaDataReader(Pipeline._current_pipeline._handle,
                           *(kwargs_pybind_reader.values()))
-
+    RocalShardingInfo = b.RocalShardingInfo()
```

Comment:
please add these defaults into constructor of RocalShardingInfo()

Reply:
Done